### PR TITLE
Add deploy frankurt {dev,stage,prod} jobs to .gitlab-ci.yml

### PR DESCRIPTION
**Pull request by jgmize**

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -22,6 +22,36 @@ deploy frankfurt:
...
+
+deploy frankfurt stage:
+  variables:
+    KUBECONFIG: /home/gitlab-runner/.kube/frankfurt.kubeconfig
+  tags:
+    - aws
+    - frankfurt
+    - meao
+  only:
+    - master
```
Comment (jpetto) [inline .gitlab-ci.yml:36]: Do we want changes to `master` to update both dev and stage environments?

File: .gitlab-ci.yml [hunk 2]
```diff
@@ -22,6 +22,36 @@ deploy frankfurt:
...
+
+deploy frankfurt prod:
+  variables:
+    KUBECONFIG: /home/gitlab-runner/.kube/frankfurt.kubeconfig
+  tags:
+    - aws
+    - frankfurt
+    - meao
+  only:
+    - master
```
Comment (jpetto) [inline .gitlab-ci.yml:51]: Same here - do we want `master` updating prod?

File: .gitlab-ci.yml [hunk 3]
```diff
@@ -22,6 +22,36 @@ deploy frankfurt:
...
+deploy frankfurt stage:
+  variables:
+    KUBECONFIG: /home/gitlab-runner/.kube/frankfurt.kubeconfig
+  tags:
+    - aws
+    - frankfurt
+    - meao
+  only:
+    - master
+    - frankfurt
```
Comment (jpetto) [inline .gitlab-ci.yml:37]: I'm guessing having this branch update dev, stage, and prod is temporary for testing?

**Review by jpetto — CHANGES_REQUESTED** [review]
A couple questions on which branches should trigger which deployments.

File: .gitlab-ci.yml (hunk 1 above)
Comment (jgmize) [reply .gitlab-ci.yml:36]: My thinking is that a push to the `master` branch of this config repo should trigger a `kubectl apply` across all environments in parallel, hence the separate jobs. For troubleshooting purposes, I've also added some optional branches we could use to target specific deployments.

File: .gitlab-ci.yml (hunk 2 above)
Comment (jgmize) [reply .gitlab-ci.yml:51]: yes, but kubectl apply here should be idempotent if there are no changes in `frankfurt/nucleus-prod`

File: .gitlab-ci.yml (hunk 3 above)
Comment (jgmize) [reply .gitlab-ci.yml:37]: The idea is that a push to the frankfurt branch would trigger parallel `kubectl apply` jobs across all the namespaces in the frankfurt cluster, but not (for example) any deployments in `iowa-b`.

File: .gitlab-ci.yml (hunk 2 above)
Comment (jpetto) [reply .gitlab-ci.yml:51]: Ah, right, right. The config files themselves will determine which apps are actually changed.

**Review by jpetto — APPROVED** [review]

**Comment by jpetto** [comment]
I think it's worth writing a bit of documentation in the readme to spell out the philosophy of the `.gitlab-ci.yaml` set up, but on board with everything here.

**Pull request merged**